packages/platform-sdk-lsk/src/services/message.ts

```diff
@@ -12,7 +12,7 @@ export class MessageService implements Contracts.MessageService {
 
 	public async sign(input: Contracts.MessageInput): Promise<Contracts.SignedMessage> {
 		try {
-			const { message, publicKey, signature } = signMessageWithPassphrase(input.message, input.mnemonic);
+			const { message, publicKey, signature } = signMessageWithPassphrase(input.message, input.mnemonic!);
```

Comment:
This needs a proper check that the mnemonic is set now that it isn't enforced by TypeScript.